test/tools.py

```diff
@@ -80,12 +82,12 @@ def _check_conda():
     # Conda can install required Python versions on demand
     _check_conda()
     HAS_CONDA = True
-except (RuntimeError, IOError) as exc:
+except (RuntimeError, IOError):
     HAS_CONDA = False
 
 
 try:
-    import virtualenv
+    import virtualenv # noqa :F401 imported but unused
```

Comment:
Same, this comment doesn't add any value, what we want is to explain why we keep this import even if it's unused, not just the flake8 error message, which doesn't provide any value to the readers of this